Author: Adam Buchbinder <adam.buchbinder@gmail.com>
Description:
 Add support for EPUB ebooks, fixing some false ZIP negatives (Closes: #596873).

diff -Naurp file.orig/magic/Magdir/archive file/magic/Magdir/archive
--- file.orig/magic/Magdir/archive	2010-09-18 11:35:32.913838992 +0200
+++ file/magic/Magdir/archive	2010-09-18 11:36:03.694842516 +0200
@@ -662,6 +662,12 @@
 >>>>78	string	-template		Template
 !:mime	application/vnd.oasis.opendocument.image-template
 
+# EPUB (OEBPS) books using OCF (Open Container Format)
+# From: Adam Buchbinder <adam.buchbinder@gmail.com>
+# URL: http://www.idpf.org/ocf/ocf1.0/download/ocf10.htm, section 4.
+>>38	string	application/epub+zip	EPUB ebook
+!:mime	application/epub+zip
+
 # Zoo archiver
 20	lelong		0xfdc4a7dc	Zoo archive data
 !:mime	application/x-zoo
